tracing.go

```diff
@@ -226,6 +229,14 @@ func (s *Span) ToSentryTrace() string {
 	return b.String()
 }
 
+func (s *Span) ToBaggage() string {
+	if dsc := s.dynamicSamplingContext; len(dsc.Entries) > 0 {
```

Comment:
Why did you choose to assign a new variable?

Reply:
Not needed, removed :)